src/js/themes/base.js

```diff
@@ -707,6 +708,30 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     //   extend: undefined,
     //   maxHeight: undefined,
     // },
+    fileInput: {
```

Comment:
Should we add an `extend` option to enable more opinionated styles?

Reply:
yep, added